client/src/views/client-area/ClientAreaPage.vue

```diff
@@ -203,10 +209,16 @@ onMounted(async () => {
     if (billingSystem === BillingSystem.CustomOrder || billingSystem === BillingSystem.ExperimentalCandidate) {
       currentPage.value = ClientAreaPages.Contracts;
       const statusResp = await BmsAccessInstance.get().getCustomOrderStatus(currentOrganization.value);
-      if (!statusResp.isError && statusResp.data && statusResp.data.type === DataType.CustomOrderStatus) {
-        if (statusResp.data.status === CustomOrderStatus.NothingLinked) {
+      const organizationStatusResp = await BmsAccessInstance.get().getOrganizationStatus(currentOrganization.value.bmsId);
+      if (!organizationStatusResp.isError && organizationStatusResp.data) {
+        statusOrganization.value = organizationStatusResp.data as OrganizationStatusResultData;
```

Comment:
You're only using `statusOrganization` locally and only to check if it's bootstraped, may as well just write
`const isBootstrapped = (!organizationStatusResp.isError && organizationStatusResp.data && organizationStatusResp.data.type === DataType.OrganizationStatus) ? organizationStatusResp.data.isBootstrap : false`

And get rid of `statusOrganization`, and just check the `isBootstrapped` variable down below.